JitsiConference.js

```diff
@@ -1160,9 +1244,11 @@ JitsiConference.prototype.getPhonePin = function () {
  * for its session.
  */
 JitsiConference.prototype.getConnectionState = function () {
-    if(this.room)
-        return this.room.getConnectionState();
-    return null;
+    if(this.jingleSession) {
+        return this.jingleSession.getIceConnectionState();
+    } else {
```

Comment:
no need, but is it so big deal ? Brian asked my for braces, so I added them everywhere. I will leave it as is if you don't mind.